server/sonar-web/src/main/js/components/issue/components/IssueTitleBar.js

```diff
@@ -74,69 +74,60 @@ export default function IssueTitleBar(props /*: Props */) {
   });
 
   return (
-    <table className="issue-table">
-      <tbody>
-        <tr>
-          <td>
-            <IssueMessage
-              message={issue.message}
-              rule={issue.rule}
-              organization={issue.organization}
+    <div className="issue-row">
+      <IssueMessage message={issue.message} rule={issue.rule} organization={issue.organization} />
```

Comment:
There is still a small quirk with the button to get rule detail, it goes alone on the next line : 
![image](https://user-images.githubusercontent.com/1725611/31650344-360c7fd0-b317-11e7-8962-7263a533599c.png)

It's not a big deal, just wanted to ping you in case you know an easy way to make it better.

Reply:
Yeah, I thought about that, but didn't find an easy way :(

Reply:
Ok, that's what I thought, It's not useful to dig into it then.